### PR TITLE
Remove default endpoint in container

### PR DESCRIPTION
**Pull request by cheungpat**
connects SkygearIO/skygear-SDK-JS#395

### CODE REVIEW

File: skygear/src/main/java/io/skygear/skygear/RequestManager.java [hunk 1]
```diff
@@ -94,6 +94,16 @@ public RequestManager(Context context, Configuration config) {
      * @param config the config
      */
     public void configure(Configuration config) {
+        if (config == null) {
```
Comment (Steven-Chan) [inline skygear/src/main/java/io/skygear/skygear/RequestManager.java]: This part looks a bit awkward.

I suggest following the behaviour in iOS. In this case setting a `null` configuration, means resetting all config. Print the warning (or maybe throw error) when trying to get configuration but the value is `null`.

Comment (cheungpat) [reply skygear/src/main/java/io/skygear/skygear/RequestManager.java]: iOS SDK does not have a Configuration object for the container.

**Comment by cheungpat** [comment]
updated, see if the newest commit works better.

**Comment by cheungpat** [comment]
hi @Steven-Chan can you take a look and see if I have addressed your concerns?

**Comment by cheungpat** [comment]
thanks!

**Pull request merged**